geode-wan/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommandDUnitTest.java

```diff
@@ -54,6 +59,63 @@ public void before() throws Exception {
     gfsh.connectAndVerify(locator);
   }
 
+  public Object[] replicatedRegionAndScopePairs() {
```

Comment:
It would be nice to add a few junit tests in `CreateRegionCommandTest`, where you can assert on the behavior if failed validation.

Reply:
Done!
Thank you for the feedback